themes/swiss21/login/template.ftl

```diff
@@ -0,0 +1,118 @@
+<#macro registrationLayout bodyClass="" displayInfo=false displayMessage=true onlyPanel=false>
+    <!doctype html>
+    <html xmlns="http://www.w3.org/1999/xhtml" class="${properties.kcHtmlClass!}">
+
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+        <meta name="robots" content="noindex, nofollow">
+
+        <#if properties.meta?has_content>
+            <#list properties.meta?split(' ') as meta>
+                <meta name="${meta?split('==')[0]}" content="${meta?split('==')[1]}">
+            </#list>
+        </#if>
+        <title>${msg("loginTitle",(realm.displayName!''))}</title>
+        <link rel="icon" href="${url.resourcesPath}/img/favicon.ico" >
+        <#if properties.styles?has_content>
+            <#list properties.styles?split(' ') as style>
+                <link href="${url.resourcesPath}/${style}" rel="stylesheet" >
+            </#list>
+        </#if>
+        <#if properties.scripts?has_content>
+            <#list properties.scripts?split(' ') as script>
+                <script src="${url.resourcesPath}/${script}" type="text/javascript"></script>
+            </#list>
+        </#if>
+        <#if scripts??>
+            <#list scripts as script>
+                <script src="${script}" type="text/javascript"></script>
+            </#list>
+        </#if>
+    </head>
+
+    <body class="${client.clientId}">
```

Comment:
This switch here might introduce problems. If you want a different style for AbaNinja you should create another theme 'abaninja', use swiss12 as parent theme, and overwrite the stuff which is relevant for AbaNinja only.

For the moment we can use the approach you have chosen. It consists of less files to manage so it is a little bit less complex to maintain.